src/commands/createNewProject.ts

```diff
@@ -177,6 +178,31 @@ async function promotForMavenParameters(ui: IUserInterface, functionAppPath: str
     };
 }
 
+async function createJavaFunctionProject(outputChannel: OutputChannel, functionAppPath: string, ui: IUserInterface): Promise<string> {
+    // Get parameters for Maven command
+    const { groupId, artifactId, version, packageName, appName } = await promotForMavenParameters(ui, functionAppPath);
+    const tempFolder: string = path.join(os.tmpdir(), Date.now().toString());
```

Comment:
Use `fsUtil.getRandomHexString()` instead of `Date.now()`

Reply:
👍